src/util/Util.js

```diff
@@ -61,12 +61,11 @@ define(['okta', './Logger', './Enums'], function (Okta, Logger, Enums) {
       return xhr;
     }
     if (!xhr.responseJSON) {
-      try {
-        xhr.responseJSON = JSON.parse(xhr.responseText);
-      } catch (parseException) {
+      if (!xhr.responseText) {
         xhr.responseJSON = { errorSummary: Okta.loc('oform.error.unexpected', 'login') };
         return xhr;
       }
+      xhr.responseJSON = xhr.responseText;
```

Comment:
**Note:** The logic of how okta-auth-js returns errors changed a little bit in the new version, this is why this change. The new version of okta-auth-js return the responseText already parsed from JSON, so we don’t need to parse it again here (which would fail, since they are not anymore in JSON format). We need instead to check if that attribute is available.